src/test/java/io/kestra/storage/s3/S3StorageTest.java

```diff
@@ -97,19 +103,35 @@ void getNoTenant() throws Exception {
 
     private void get(String tenantId, String prefix) throws Exception {
         URL resource = S3StorageTest.class.getClassLoader().getResource("application.yml");
-        String content = CharStreams.toString(new InputStreamReader(new FileInputStream(Objects.requireNonNull(resource).getFile())));
+        String content = com.google.common.io.CharStreams.toString(new InputStreamReader(new FileInputStream(Objects.requireNonNull(resource).getFile())));
```

Comment:
```suggestion
        String content = CharStreams.toString(new InputStreamReader(new FileInputStream(Objects.requireNonNull(resource).getFile())));
```

CharStreams is imported so please remove qualified name everywhere it is used